hw/xbox/nv2a.c

```diff
@@ -5768,6 +5783,15 @@ static void pgraph_method(NV2AState *d,
         NV2A_DPRINTF("------------------CLEAR 0x%x %d,%d - %d,%d  %x---------------\n",
             parameter, xmin, ymin, xmax, ymax, d->pgraph.regs[NV_PGRAPH_COLORCLEARVALUE]);
 
+        /* Dither */
```

Comment:
I haven't tested this on hardware yet. So I'm not sure.

It could also be a very glorified memset. I've only ever called it 'memset' because it's one of the very few methods which respects the surface format. 
One example when clear without dither would cause bugs is the JSRF intro screen (which is not dithered = theoretical case) which would be dithered in the center (which is a logo texture), but not dithered around it (which is a CLEAR). Also the HW is created around GL and GL *does* dither CLEARs (which means PC drivers would have had to reimplement glClear using a fullscreen blit = almost always because GL_DITHER is enabled by default).

The FIXME " Maybe also disable it here?" should be enough to remind us.